Code/Editor/AnimationContext.cpp

```diff
@@ -487,22 +466,6 @@ void CAnimationContext::Update()
         m_bForceUpdateInNextFrame = false;
     }
 
-    // If looking through camera object and recording animation, do not allow camera shake
-    if ((GetIEditor()->GetViewManager()->GetCameraObjectId() != GUID_NULL) && GetIEditor()->GetAnimation()->IsRecording())
-    {
-        if (GetIEditor()->GetMovieSystem())
-        {
-            GetIEditor()->GetMovieSystem()->EnableCameraShake(false);
-        }
-    }
-    else
-    {
-        if (GetIEditor()->GetMovieSystem())
-        {
-            GetIEditor()->GetMovieSystem()->EnableCameraShake(true);
-        }
-    }
```

Comment:
this was easy to remove the camera shake is unused code so its better to just remove it then keep the logic faulty.